### ISSUE TITLE
"No storage mount" on latest Manjaro Linux (Arch-based)

### ISSUE THREAD

**Issue by TKCZ** [BoostIO/Boostnote#2633]
# Current behavior

When I start Boostnote, it says "No storage mount." right under Trash. Adding it via Prefences (different Storage) doesn't change the situation. Can't even create a note.

# Expected behavior

I expect to be able to create storage and mount it. Right now all I get is "No storage mount." therefore I cannot create not even a single note.

# Steps to reproduce

1. Start Boostnote 
2. Try to create note; if you can't ("No storage to create a note") try checking under Trash
3. Check in Preferences if adding another Storage point changes the situation in any way

# Environment

- Version : current latest 0.11.11
- OS Version and name : Manjaro Linux KDE; Linux tom-pc 4.19.2-1-MANJARO #1 SMP PREEMPT Tue Nov 13 22:35:25 UTC 2018 x86_64 GNU/Linux

I don't know about inner workings of Boostnote, but similair issue with mounting drives happened in Cryptomator for me. There the issue was somehow connected to change of packages (from gvfs-mount to simply gvfs) in latest upstream, therefore only Arch seems to be affected from Linux distros so far. But this definitelly needs confirmation as I am definitelly eye-balling this and may not be related at all.

**Comment by TKCZ** [comment]
The issue with Cryptomator I mentioned:

https://community.cryptomator.org/t/connecting-drive-failed-on-windows-7/2327/7

**Comment by ZeroX-DG** [comment]
Can you open the dev tool and show us the error? `ctrl + shift + i` is the hotkey for toggling dev tool.

**Comment by TKCZ** [comment]
Yes of course!

`elixir.js Failed to load resource: net::ERR_FILE_NOT_FOUND
main.js:1360 boostnote.json file doesn't exist the given path
resolveStorageData @ main.js:1360
/usr/lib/boostnote/node_modules/mkdirp/index.js:90 Uncaught Error: EEXIST: file already exists, mkdir '/run/user/1000/gvfs'
    at Object.fs.mkdirSync (fs.js:885)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:71)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:77)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
    at sync (/usr/lib/boostnote/node_modules/mkdirp/index.js:78)
/usr/lib/electron/resources/electron.asar/renderer/security-warnings.js:188 Electron Security Warning (Insecure Content-Security-Policy) This renderer process has either no Content Security Policy set or a policy with "unsafe-eval" enabled. This exposes users of this app to unnecessary security risks.

For more information and help, consult https://electronjs.org/docs/tutorial/security.
This warning will not show up once the app is packaged.
warnAboutInsecureCSP @ /usr/lib/electron/resources/electron.asar/renderer/security-warnings.js:188`

I also managed to make a screenshot to make sure:

https://i.imgur.com/yt0tpZx.png

**Comment by TKCZ** [comment]
So, I still do not know the root cause of this issue, but I finally have a workaround. On my current Manjaro KDE I did:

1) Completely Remove Boostnote via Package manager like octopi/pamac preferably (or any other way really) & node-js to make sure..

2) Then make sure you also delete folder Boostnote in your ~/.config/ 

3) Then I ran "sudo pacman -Rsn $(pacman -Qdtq)" to get rid of old packages and mess..

4) Finally I installed Boostnote again as usual and it works - so issue must have been somewhere in config files it seems. This "complete wipe" did work after all so I consider this issue as resolved.
